app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+	belongs_to :author, class_name: 'User'
```

Comment:
we had a validation for presence on author already, but I added another one to commentable so we can ensure that every comment is associated with an idea or primary comment. 
[development 76e1191]